src/helpers.js

```diff
@@ -12,8 +12,8 @@ export const execShellCommand = (cmd) => {
       spawn("C:\\msys64\\usr\\bin\\bash.exe", ["-lc", cmd], {
         env: {
           ...process.env,
+          "HOME": process.env.GITHUB_WORKSPACE,
```

Comment:
I don't think that's a good idea: that's not the home directory, really, therefore debugging could become unnecessarily confusing e.g. when the workspace contains a `.gitconfig`, or when the workflow-to-debug installed something into the _actual_ home directory and that caused the failures.

I am afraid that we have to continue using that ugly `CHERE_INVOKING` feature.

Reply:
valid point! This would definitely introduce edge-cases that it behaves different to the actual GitHub runner.

I implemented again something else which would work, let me know what you think.